src/plugin/range/index.js

```diff
@@ -0,0 +1,57 @@
+
+export class DayjsRange {
+  constructor(startDate, endDate) {
+    const { s, e } = startDate > endDate
+      ? { s: endDate, e: startDate }
+      : { s: startDate, e: endDate }
+
+    this.startDate = s
+    this.endDate = e
+  }
+
+  isValidRange() {
+    return this.startDate.isValid() && this.endDate.isValid()
+  }
+
+  clone() {
+    return new DayjsRange(this.startDate, this.endDate)
+  }
+
+  isOverlap(other) {
```

Comment:
how about an option for `isOverlap` to exclude equals?

A from: 16:00
A to: 17:00

compared to 
B from: 17:00
B to: 18:00 

shouldn't return `true` in my opinion when dealing with calendar entries
compared